src/main/java/com/google/jenkins/plugins/storage/AbstractUpload.java

```diff
@@ -93,14 +93,23 @@
 // TODO: Refactor to use StorageClient https://github.com/jenkinsci/google-storage-plugin/issues/71
 public abstract class AbstractUpload
     implements Describable<AbstractUpload>, ExtensionPoint, Serializable {
-
   private static final Logger logger = Logger.getLogger(AbstractUpload.class.getName());
   private static final ImmutableMap<String, String> CONTENT_TYPES =
       ImmutableMap.of(
           "css", "text/css",
           "js", "application/javascript",
           "svg", "image/svg+xml",
           "woff2", "font/woff2");
+  private String pathPrefix;
+  /** The module to use for providing dependencies. */
+  protected final transient UploadModule module;
+  /** Provide detail information summarizing this download for the GCS upload report. */
+  public abstract String getDetails();
+  /** NOTE: old name kept for deserialization */
```

Comment:
Nit: Given that this is private, this doesn't need to be javadoc, `// ...` or `/* ... */` would be fine